go.mod

```diff
@@ -2,4 +2,7 @@ module github.com/medic/cht-docs
 
 go 1.13
 
-require github.com/google/docsy v0.7.0 // indirect
+require (
+	github.com/google/docsy v0.10.0 // indirect
+	github.com/google/docsy/dependencies v0.7.0 // indirect
```

Comment:
Shouldn't these versions match?

Reply:
For some reason it keeps reverting to 0.7.2 when I run hugo server. The error is: `go: github.com/google/docsy/dependencies@v0.10.0: invalid version: unknown revision dependencies/v0.10.0
 *errors.errorString`